pkg/packerjanitor/packer_janitor.go

```diff
@@ -0,0 +1,242 @@
+package packerjanitor
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+	"go.uber.org/zap"
+
+	"time"
+)
+
+const (
+	// RFC8601 is the date/time format used by AWS.
+	RFC8601 = "2006-01-02T15:04:05.000Z"
+	// DryRun is the type of error thrown by AWS when a task fails
+	// because it was run with the DryRun option but would have
+	// otherwise succeeded.
+	DryRun = "DryRunOperation"
+)
+
+// PackerClean is a generic struct used for the various functions.
+type PackerClean struct {
+	Delete         bool
+	ExpirationDate time.Time
+	Logger         *zap.Logger
+	EC2Client      ec2iface.EC2API
+}
+
+// GetPackerInstances -- find all running instances that are Packer
+// builds older than X and returns them in a list
+func (p *PackerClean) GetPackerInstances() ([]*ec2.Instance, error) {
+	var output *ec2.DescribeInstancesOutput
```

Comment:
Is this needed here? On line 42 you're doing the regular ```:=``` assignment, which should cover it.

Reply:
Hrm, I think I was getting complaints from go vet when I wasn't doing that before for some reason...it might have been because of something else I was doing elsewhere though. I can pull it out.